src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -266,6 +267,7 @@ protected override SyntaxNode FindStatementAndPartner(SyntaxNode declarationBody
 
                 // No need to special case property initializers here, the active statement always spans the initializer expression.
 
+                RoslynDebug.Assert(declarationBody.Parent is object);
```

Comment:
>. [](start = 27, length = 1)

I think it'd be better to consistently use Parent! rather than adding asserts in some places and ! in others.